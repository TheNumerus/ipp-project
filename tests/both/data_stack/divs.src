.ippcode20
PUSHS float@0x1.4p+2
pushs float@0x1.8p+1
divs
defvar GF@temp
pops GF@temp
write GF@temp