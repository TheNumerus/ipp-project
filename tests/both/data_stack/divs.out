0x1.aaaaaaaaaaaabp+0